packages/pico8-ls/package.yaml

```
@@ -0,0 +1,20 @@
+---
+name: pico8-ls
+description: PICO-8 Language Server
+homepage: https://github.com/japhib/pico8-ls
+licenses:
+  - MIT
+languages:
+  - PICO-8
+  - PICO-8 Lua
+categories:
+  - LSP
+source:
+  # renovate:datasource=git-refs
+  id: pkg:github/japhib/pico8-ls
```

A version specifier is always necessary, let's use latest `HEAD`:

```suggestion
  id: pkg:github/japhib/pico8-ls@ef01bb52cba6fe358dcc2478ace74033744fdd26
```